vector/v.surf.rst/v.surf.rst.html

```diff
@@ -293,6 +293,24 @@ <h3>Cross validation procedure</h3>
 with lost details and fluctuations) or when significant noise is
 present that needs to be smoothed out.
 
+<h3>Performance</h3>
+To enable parallel processing, the user can specify the number of threads to be
+used with the <b>nprocs</b> parameter (default 1). Figures 1 and 2 show benchmark results
+running on Intel® Core™ i5-10210U CPU @ 1.60GHz × 8. See benchmark scripts
+in the source code for more details.
+
+<div align="center" style="margin: 10px">
+  <img src="vsurfrst_benchmark.png" alt="benchmark for v.surf.rst" border="0">
+  <br>
+  <i>Figure 1: Benchmark shows execution time for different
+    number of cells (1M, 2M, 4M, and 8M).
+</div>
+<div align="center" style="margin: 10px">
+    <img src="vsurfrst_cv_benchmark.png" alt="benchmark for cross-validation of v.surf.rst" border="0">
+    <br>
+    <i>Figure 2: Benchmark shows execution time for running cross-validation on different number of cells (100k, 200k, 400k, and 800k).
```

Comment:
Please break long lines in the HTML file as per our style guide:

https://github.com/OSGeo/grass/blob/main/doc/development/style_guide.md#markup-style-guide

More notes on markup:
- In order to minimize potential git merge conflicts, please break a line at approximately 70-80 chars.

Reply:
fixed in https://github.com/OSGeo/grass/pull/3590/commits/c7cbf386c97a2a60f0b7319addc54d811e5b5a16